gips/0081-indexing-payments.md

```diff
@@ -0,0 +1,165 @@
+Replaces: GIP-0058
+---
+
+## Abstract
+
+This GIP describes **Indexing Payments**, a mechanism by which a protocol participant (commonly a gateway) can incentivize a candidate indexer to service a subgraph. It details a target, trust-minimized end-state that relies on Graph Horizon, while also introducing a semi-trusted Minimum Viable Product (MVP) based on the current protocol and off-chain components (gateway + indexer stacks). This GIP is an alternative, simplified proposal to [GIP-0058](https://github.com/graphprotocol/graph-improvement-proposals/blob/main/gips/0058-Replacing-Bonding-Curves-with-Indexing-Fees.md).
+
+## Motivation
+
+Curation has generally been perceived as a source of friction for Developers that may only want to have subgraphs indexed to power their decentralized applications. It requires locking up certain amounts of GRT, and the necessary amounts might vary depending on curation on other subgraphs. While the quality of service provided on the network is generally high, Indexers collect rewards for posting Proofs of Indexing independently to the query service they provide, so Developers are not guaranteed a certain level of service for the amount of curation they put into the system.
+
+For Indexers, there is no clear relationship between the Indexing Rewards on a subgraph and the amount of work required to index it. Attempts at defining better curation and rewards models to pay for indexing have so far been unsuccessful, which presents a need for a simpler mechanism where parties interested in indexing a subgraph can simply pay for it through the protocol. However, when building such a mechanism, it is imperative to preserve the protocol's trust-minimization and decentralization principles.
+
+## Prior Art
+
+GIP-0058 presents a previous version of a very similar proposal. This GIP re-introduces the same concepts, but incorporates the latest approach in Graph Horizon, and simplifies the proposal with an off-chain MVP while clarifying that the actor engaging in **Indexing Payments** is generally a Gateway running automated indexer selection.
+
+## High-Level Description
+
+We are proposing a new coordination mechanism to incentivize subgraph serving to a performant level. Such a coordination mechanism is necessary to jump-start, and sometimes maintain, performant subgraph serving. One such time is when a subgraph is being upgraded. In early conversations this has been referred to as **Direct Indexing Payments** (DIPs).
+
+In particular, the coordination mechanism has been designed with the following in mind:
+
+- The objective is to get subgraphs served through the protocol to a sufficiently performant level
+- The price to get a subgraph served at said level should ideally be related to the amount of work required.
+
+While the proposal is meant to fully support this use case, it does not exclude other use cases that may organically arise. Several components are general and can be re-used.
```

Comment:
'general' -> 'generic'

Reply:
organically arise could be arise organically

Reply:
done